omniauth_login_dot_gov.gemspec

```
@@ -29,7 +29,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'json-jwt', '~> 1.11'
   s.add_dependency 'jwt', '~> 2.2'
   s.add_dependency 'multi_json', '~> 1.14'
-  s.add_dependency 'omniauth', '~> 1.8.1'
+  s.add_dependency 'omniauth', '~> 2.0.4'
```

are we depending on anything specific? I suspect this is a little too precice (ex a `2.1` would need a similar PR), maybe we just say  `> 1.0.0` or something? Or prevent major version bumps which might contain API breaks:

```suggestion
  s.add_dependency 'omniauth', '< 3.0.0'
```

---

Yeah let's go with your suggestion @mitchellhenke

---

this change has been made